libs/coin-bitcoin/src/mockBtc.ts

```diff
@@ -1,18 +1,19 @@
-import Btc, { AddressFormat } from "@ledgerhq/hw-app-btc";
-import { CreateTransactionArg } from "@ledgerhq/hw-app-btc/createTransaction";
-import Transport from "@ledgerhq/hw-transport";
 import hash from "object-hash";
+import type {
+  AddressFormat,
+  BitcoinSignature,
+  BitcoinSigner,
+  BitcoinXPub,
+  CreateTransaction,
+  SignerTransaction,
+} from "./signer";
 
-class MockBtc extends Btc {
-  constructor() {
-    super({ transport: new Transport() });
-  }
-
+class MockBtc implements BitcoinSigner {
   // eslint-disable-next-line class-methods-use-this
   async getWalletPublicKey(
     path: string,
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    opts: { verify?: boolean; format?: AddressFormat },
+    _opts: { verify?: boolean; format?: AddressFormat },
```

Comment:
as we've hard coded the return "chaincode" and "publickey" in the mock method. this "_opts" parameter can be deleted and it is not used.